install/kubernetes/cilium/templates/cilium-agent-clusterrole.yaml

```diff
@@ -104,6 +104,7 @@ rules:
   - ciliumlocalredirectpolicies
   - ciliumlocalredirectpolicies/status
   - ciliumlocalredirectpolicies/finalizers
+  - ciliumegressnatpolicies
```

Comment:
We need `ciliumlocalredirectpolicies/status`. I'm not so sure about `ciliumlocalredirectpolicies/finalizers`. But we should probably put it to be on the safe side?

Reply:
ciliumegressnatpolicies doesn't have `status` field. but I guess there is no harm adding it though. will follow the pattern here.

Reply:
@anfernee if that is the case then we should remove it from the CRD as it is currently defined in there https://github.com/cilium/cilium/blob/a783f7c55d34b647e751afe6fc7e892b1a809485/examples/crds/ciliumegressnatpolicies.yaml#L184-L185
https://github.com/cilium/cilium/blob/a783f7c55d34b647e751afe6fc7e892b1a809485/pkg/k8s/apis/cilium.io/v2alpha1/types.go#L27